### PR TITLE
feat: show hotkeys

### PR DESCRIPTION
**Pull request by edinapap**
Adding a basic list of hotkeys for the user

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,17 @@
...
+		<div class="hotkeys">
+			Hotkeys: <br/><br/>
+			Space: apply selected colour <br/>
+			1: Camera view 1 <br/>
+			2: Camera view 2 <br/>
+			R: Reset handles <br/>
+			X: Toggle X handles <br/>
+			Y: Toggle Y handles <br/>
+			Drag and press Shift: Toggle separated handles <br/>
+			</span>
```
Comment (webadam1) [inline index.html:19]: This closing tag is not needed

File: style.css [hunk 2]
```diff
@@ -12,6 +12,12 @@ body {
   justify-content: center;
 }
 
+.hotkeys {
```
Comment (webadam1) [inline style.css:15]: We might want to make it only show on hover, because on smaller screens it overlays the gradient mesh

Comment (webadam1) [reply style.css:15]: Or just toggleable with a button or something

**Review by webadam1 — DISMISSED** [review]
Pretty much what we need, but having it only show when the user wants to see it would be great.

File: style.css (hunk 2 above)
Comment (edinapap) [reply style.css:15]: Good idea

File: index.html (hunk 1 above)
Comment (edinapap) [reply index.html:19]: nice catch, thanks

**Review by webadam1 — APPROVED** [review]

**Pull request merged**